src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/instrument-lambda-function/enable-serverless-monitoring-aws-lambda-layerless.mdx

```diff
@@ -169,8 +170,22 @@ If you're using a Lambda function without a layer, you'll need to instrument New
          callback();
        });
        ```
+       - Sample Code for ES Module:
+
+       ```js
+          import newrelic from 'newrelic';
+          // Other module loads go under the require statement above
+
+          export const handler = newrelic.setLambdaHandler((event, context, callback) => {
+          // This is your handler function code
+          console.log('Lambda executed');
+          callback();
+        });
+       ```
 
     3. Optional: You can also add [custom events](/docs/using-new-relic/welcome-new-relic/getting-started/glossary#custom-event) to your Lambda using the [`recordCustomEvent` API](/docs/agents/nodejs-agent/api-guides/nodejs-agent-api#record_custom_event). For example:
+      
+       - Sample code for CommonJS:
```

Comment:
Suggestion: use "*" instead of "-" for bulleted list.
 example:  * Sample Code for CommonJS: